src/main/java/org/fundacionjala/sfdc/pageobjects/BasePage.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.util.CommonActions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * BasePage.java
+ * Class to initialize WebDriver for all pages.
+ */
+public abstract class BasePage {
+    protected WebDriver webDriver;
+    protected WebDriverWait wait;
+    protected CommonActions commonActions;
+
+    /**
+     * Constructor.
+     */
+    public BasePage() {
+        this.webDriver = DriverManager.getNavigator();
+        this.wait = DriverManager.getWaiter();
+//        this.commonActions = new CommonActions();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)